website/documentation/050-tutorials/content/howto/insecure-configuration/_index.md

```diff
@@ -13,48 +13,102 @@ aliases: ["readmore/insecure"]
 
 {{% howto_img "" "./images/teaser.svg" %}}
 
-## Introduction
-In the last summer, the [Gardener team](https://github.com/gardener/gardener) worked on assessing the security of 
-different Gardener managed Kubernetes installations. **The goal of this ongoing work is to increase the security of 
-all Gardener stakeholders in the Open Source community.** Following the Gardener architecture, the control plane of a 
-Gardener managed shoot cluster resides in the corresponding seed cluster. This is a 
-[Control-Plane-as-a-Service](https://kubernetes.io/blog/2018/05/17/gardener/#kubernetes-control-plane) with 
-a [network air gap](https://kubernetes.io/blog/2018/05/17/gardener/#network-air-gap).
+## Goal: Increasing the Security of all Gardener Stakeholders
```

Comment:
you changed the title. 
but the text (except for the emphasis **) is the same. imho the 80 char line breaks were fine.

Reply:
Sorry about the line breaks. How can I take back my pull request? I think I need to start all over again if the line breaks are an issue - I see my changes also in Word so I can do it line by line instead of paragraph by paragraph.

imho, using bold text in the text body should be avoided for better readability. The title "Introduction" was quite generic, so my suggestion would be to change the title to reflect the importance of the aspect previously formatted in bold.

Reply:
Hi @vasu1124 , the changes should be easier to review now.

Reply:
@vasu1124 @freegroup 
About the images: Please have a look at the current [online version](https://gardener.cloud/050-tutorials/content/howto/insecure-configuration/).

Scroll down to section "Scenario 2: ...".
The images contain boxes for "grafana" and "prometheus". 
However, this is discussed in scenario 3, not scenario 2.

That's why I assume that the images do not fit and were swapped by mistake.